### ISSUE TITLE
update readme in tools/torch please

### ISSUE THREAD

**Issue by mastrcheif** [NVIDIA/DIGITS#887]
Well I tried to follow the example in https://github.com/NVIDIA/DIGITS/tree/master/tools/torch and get the error that --load can not be found and looking deeper into https://github.com/NVIDIA/DIGITS/blob/master/tools/torch/test.lua there is no --load required Now I am on a path of trail and error to see what works and what not :/

**Comment by mastrcheif** [comment]
Well, this is the command that worked for me for getting any predictions

`th test.lua \
--networkDirectory ./pretrained_model/ \
--snapshot ./pretrained_model/snapshot_30_Model.t7 \
--image ./val/honey/frame25_192.168.1.122.jpg \
--network ./pretrained_model/resnet.lua \
 --labels ./pretrained_model/labels.txt \
--mean ./pretrained_model/mean.jpg \
--subtractMean pixel
`

**Comment by mastrcheif** [comment]
But why am I getting total different results on the same network and on the same picture. The web app 'classidy one' method tells me to 98% that it is product A and the https://github.com/NVIDIA/DIGITS/blob/master/tools/torch/test.lua file is telling me that it is product B with 38 %

**Comment by gheinrich** [comment]
Hi @mastrcheif sorry that part of the doc is out of sync with the latest code. The snapshot loading scheme was updated and now the full snapshot path is passed with `--snapshot`.  Parameter `--load` was removed.

I have updated the doc in #892. Can you verify please?

I have verified that I am getting the same probabilities on the command line and on the web UI.
Things to check on your side:
- make sure the input image has exactly the dimensions that your network expects (DIGITS resizes images before passing them to `test.lua` so you will need to do the same thing in your script)
- make sure you are using the same mean subtraction scheme (pixel, image, ...)
- make sure you are using exactly the same version of Torch for training and inference
- if you are using batch normalization and training on GPU, double check that you are also doing inference on GPU

**Comment by mastrcheif** [comment]
Hey @gheinrich thanks for your help, alright it works now. I just had to scale down my image to Resnet's 224 x 224 and it gives me the same results. I thought resizing was taking care by the test.lua file, but the python layer is taking care of it, as it seems.

One litte addition, the command also works without the wrapper.lua part just as fine
and thanks again =)
